lib/src/native/realm_core.dart

```diff
@@ -843,12 +849,44 @@ class _RealmCore {
     });
   }
 
-  AppHandle getApp(ApplicationConfiguration configuration) {
-    final httpTransport = createHttpTransport(configuration.httpClient);
-    final appConfig = createAppConfig(configuration, httpTransport);
-    final syncClientConfig = createSyncClientConfig(configuration);
-    final realm_app = _realmLib.invokeGetPointer(() => _realmLib.realm_app_get(appConfig._pointer, syncClientConfig._pointer));
-    return AppHandle._(realm_app);
+  AppHandle getApp(AppConfigHandle appConfigHandle, SyncClientConfigHandle syncClientConfigHandle) {
```

Comment:
Let's discuss this at the roundtable, but I think we should return the previous form of this method.

Reply:
I have added an agenda item